arangod/Aql/AstResources.cpp

```diff
@@ -65,12 +88,13 @@ size_t AstResources::newCapacity(T const& container,
 
   size_t capacity = container.size() + 1;
   if (capacity > container.capacity()) {
-    capacity *= 2;
+    capacity = container.size() * 2;
```

Comment:
No, in line 91, `size()` can newer be 0. This is because there is a check for container emptiness in line 84 (directly before). So `size()` is at least one here, so that doubling it will always have an effect.

Reply:
Good. I had not seen this.